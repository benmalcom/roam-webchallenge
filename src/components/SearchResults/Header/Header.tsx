import React from 'react';
import { Link } from 'react-router-dom';
import './Header.scss';
import Aux from '../../_App/partials/Aux';

export default () => (
	<Aux>
		<h4 className="text-muted">
			Search results &nbsp; &nbsp;
			<Link to="/" className="btn btn-sm btn-outline-info text-sm text-right">
				<i className="fa fa-arrow-left"/> Back to search</Link>
		</h4>
		<hr/>
	</Aux>
);
