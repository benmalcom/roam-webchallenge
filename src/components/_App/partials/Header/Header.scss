nav{
  box-shadow: 0 0 1px 0 hsla(0,0%,0%,0.2);
}
