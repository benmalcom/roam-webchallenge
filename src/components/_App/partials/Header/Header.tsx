import React from 'react';
import { NavLink } from 'react-router-dom';
import './Header.scss';
export default () => (
	<nav className="navbar navbar-expand-lg bg-white fixed-top guest-navbar">
		<NavLink to="/" className="navbar-brand text-info text-muted">
			<i className="fa fa-map-marker text-danger"/> Spot Finder</NavLink>
		<button
			className="navbar-toggler"
			type="button" data-toggle="collapse"
			data-target="#navbarSupportedContent"
			aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
			<span className="fa fa-bars text-white" />
		</button>

		<div className="collapse navbar-collapse align-items-center" id="navbarSupportedContent"/>
	</nav>
);
