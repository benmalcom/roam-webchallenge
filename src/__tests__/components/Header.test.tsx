import React from 'react';
import Header from '../../components/_App/partials/Header/Header';
import { shallow } from 'enzyme';
describe('<Header />', () => {
	test('Component renders properly', () => {
		const component = shallow(<Header />);
		expect(component).toMatchSnapshot();
	});
});
