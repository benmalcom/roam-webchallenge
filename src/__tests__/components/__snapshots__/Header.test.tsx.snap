// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Component renders properly 1`] = `
<nav
  className="navbar navbar-expand-lg bg-white fixed-top guest-navbar"
>
  <NavLink
    activeClassName="active"
    aria-current="page"
    className="navbar-brand text-info text-muted"
    to="/"
  >
    <i
      className="fa fa-map-marker text-danger"
    />
     Spot Finder
  </NavLink>
  <button
    aria-controls="navbarSupportedContent"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#navbarSupportedContent"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="fa fa-bars text-white"
    />
  </button>
  <div
    className="collapse navbar-collapse align-items-center"
    id="navbarSupportedContent"
  />
</nav>
`;
